include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3052,6 +3051,34 @@ struct OrtApi {
   *             Only use it for custom kernel launching.
   */
   ORT_API2_STATUS(KernelContext_GetGPUComputeStream, _In_ const OrtKernelContext* context, _Outptr_ void** out);
+
+  /// @}
+  /// \name GetTensorMemoryInfo
+  /// @{
+  /**
+   * \brief Get a pointer to the requested version of the Execution Provider specific
+   * API extensions to the OrtApi 
+  * \param[in] ort_value ::OrtValue containing tensor.
+  * \param[out] mem_info ::OrtMemoryInfo of the tensor. Do NOT free the returned pointer. It is valid for the lifetime of the ::OrtValue
+  */
```

Comment:
Missing the OrtStatus documentation line here

Reply:
fixed